apps/web/app/composables/usePayPal/usePayPal.ts

```diff
@@ -134,9 +154,14 @@ export const usePayPal = () => {
     state.value.isReady = false;
     state.value.paypalScript = null;
     state.value.loadingScripts[scriptKey] = loadScript(currency, localePayPal, commit)
-      .then((paypalScript) => {
+      .then(async (paypalScript) => {
         state.value.paypalScript = { script: paypalScript, currency, locale: localePayPal, commit };
         state.value.isReady = true;
+
+        if (paypalScript) {
+          await updateAvailableAPMs(paypalScript, currency);
```

Comment:
nice that we do that now with with the script. There is a noticeable delay now when waiting for the paypal buttons tho